internal/scene/sceneOneBeach.md

```diff
@@ -0,0 +1,43 @@
+# beginning
+`[Audio: Wave.ogg]`
+
+You open your eyes.
+
+You find yourself at a beach. You hear the waves come and go, the red sunset reflects on the water’s surface.
+
+As the sunlight falls, a shiny reflection catches your eye.
+
+`(Inspect reflection) > get_compass`
+
+# get_compass
+You walk closer to whatever it is that caught your eye. It was glass that reflected sunlight into your eyes. Glass that belonged to a little device. A compass.
+
+`(Pick up compass) > got_compass`
+
+# got_compass
+`[Audio: Harp.ogg]`
```

Comment:
The Audio change to the song in here overwrites the waves, which kind of took me out of the moment. Would be nicer to have the waves continuing in the background while we're in this scene, but we can add that in a future update and merge this first

Reply:
I found the harpies rather intriguing in that moment ;-). But as it's only a placeholder as I understood it, I've just wanted to check if the audio actually switched over to a new track.